qstnrs/questionnaire/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+
+
+class Questionnaire(models.Model):
+    questionnaire_name = models.CharField(max_length=150)
+    questionnaire_description = models.TextField(max_length=500)
+
+    def __unicode__(self):
+        return self.questionnaire_name[:100]
+
+    class Meta:
+        ordering = ['questionnaire_name']
+
+
+class Page(models.Model):
+    questionnaire = models.ForeignKey(Questionnaire)
+    page_name = models.CharField(max_length=50)
+    page_order = models.IntegerField()
+
+    def __unicode__(self):
+        return self.page_name
+
+
+class Question(models.Model):
+    page = models.ForeignKey(Page)
+    question_text = models.CharField(max_length=250)
+    question_order = models.IntegerField()
+
+    def __unicode__(self):
+        return self.question_text[:100]
+
+    class Meta:
```

Comment:
I will use that and write a schema migration to remove the page_order and question_order fields.